packages/runtime/test/src/runtime.ts

```diff
@@ -286,4 +302,281 @@ describe("Algorand Standard Assets", function () {
     assert.equal(john.getAssetHolding(assetId)?.amount, 0);
     assert.equal(alice.getAssetHolding(assetId)?.amount, initialAliceAssets + initialJohnAssets);
   });
+
+  it("should throw error if asset is not found while modifying", () => {
+    const modifyParam: ModifyAssetParam = {
+      type: TransactionType.ModifyAsset,
+      sign: SignType.SecretKey,
+      fromAccount: john.account,
+      assetID: 120,
+      fields: modFields,
+      payFlags: {}
+    };
+    expectTealError(
+      () => runtime.executeTx(modifyParam),
+      ERRORS.ASA.ASSET_NOT_FOUND
+    );
+  });
+
+  it("should modify asset", () => {
+    const assetId = runtime.createAsset('gold',
+      { creator: { ...john.account, name: "john" } });
```

Comment:
I think we are using this in each test. How about putting this in `beforeEach()` hook ?

Reply:
yes